src/components/room-invitation-creation-modal.yml

```diff
@@ -0,0 +1,6 @@
+newRoomInvitation:
+  en: New invitation
+  de: Neuer Einladung
+emailRequired:
+  en: Please specify an Email
+  de: Bitte geben Sie einen E-Mail an
```

Comment:
eine E-Mail-Adresse

Reply:
Fixed, I left the "an", not sure if that's okay

Reply:
Ah yes, we need the "an" :-)